Assets/HoloToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -78,24 +117,31 @@ private void Update()
                 if (Physics.Raycast(headPosition, gazeDirection, out hitInfo,
                     30.0f, spatialMappingManager.LayerMask))
```

Comment:
Could we also either put everything on one line, or a new line after each comma?